flow/Trace.cpp

```diff
@@ -550,6 +550,8 @@ struct TraceLog {
 		}
 	}
 
+	void setLogGroup(const std::string& logGroup) { this->logGroup = logGroup; }
```

Comment:
The `logGroup` variable could be being accessed from a different thread, so we probably should guard this with the trace mutex.